src/service/resource.js

```diff
@@ -265,7 +265,7 @@ export class Resource {
       this.state_ = ResourceState.NOT_LAID_OUT;
     }
     if (opt_callback) {
```

Comment:
I don't get your point here. `Resource#changeSize` needs the `opt_callback` or `callback` to resolve the `Resources#attemptChangeSize` promise. 
The case that `Resourse#changeSize` deosn't need an opt_callback is when `Resources#changeSize` is called. e.g.
https://github.com/ampproject/amphtml/blob/master/src/custom-element.js#L1287